app/views/_logo.html.erb

```diff
@@ -1,4 +1,3 @@
 <a id="logo" class="navbar-brand" href="<%= hyrax.root_path %>" data-no-turbolink="true">
-  <span class="glyphicon glyphicon-globe" role="img" aria-label="<%= application_name %>" aria-hidden="true"></span>
-  <span class="institution_name"><%= application_name %></span>
+  <%= image_tag("hyrax_logo-bw.png", alt: application_name) %>
```

Comment:
perhaps this could be done as something configurable such as https://github.com/samvera/hyrax/blob/5a9d1be16ee1a9150646384471992b03aab527a5/app/views/layouts/homepage.html.erb#L3

Reply:
I'm going to pull the logo out of this PR.